charts/rucio-webui/README.md

```diff
@@ -0,0 +1,168 @@
+# Rucio
+
+##  Data Management for science in the Big Data era.
+
+Rucio is a software framework that provides functionality to organize, manage, and access large volumes of scientific data using customisable policies. The data can be spread across globally distributed locations and across heterogeneous data centers, uniting different storage and network technologies as a single federated entity. Rucio offers advanced features such as distributed data recovery or adaptive replication, and is highly scalable, modular, and extensible. Rucio has been originally developed to meet the requirements of the high-energy physics experiment ATLAS, and is continuously extended to support LHC experiments and other diverse scientific communities.
+
+## QuickStart
+
+Add the Rucio Helm repository to your local Helm installation:
+```bash
+helm repo add rucio https://rucio.github.io/helm-charts
```

Comment:
this link returns 404 at the moment https://rucio.github.io/helm-charts - not an issue but just wanted to point it out

Reply:
It's okay. The file of interest is not index.html but index.yaml
https://rucio.github.io/helm-charts/index.yaml